cob_twist_controller/include/cob_twist_controller/cob_twist_controller_data_types.h

```diff
@@ -59,6 +64,16 @@ enum ContraintTypes {
     GPM_JLA_MID = 4,
     GPM_CA = 5,
 };
+struct ExtendedJacobianDimension {
+        ExtendedJacobianDimension():
+            lin_x(0),lin_y(0),lin_z(0),rot_x(0),rot_y(0),rot_z(0){}
+    double lin_x;
+    double lin_y;
+    double lin_z;
+    double rot_x;
+    double rot_y;
+    double rot_z;
+};
```

Comment:
also, I think this should go into a new class together with a "process"-function that handles the processing of the values from the q_dot_ik that correspond to the additional dof's, i.e. for the base this would be the publisher to the twist topic...